k8s/tikv/README.md

```diff
@@ -0,0 +1,370 @@
+
+```
+kubectl patch svc "$APP_INSTANCE_NAME-pd" \
+  --namespace "$NAMESPACE" \
+  --patch '{"spec": {"type": "LoadBalancer"}}'
+```
+
+> **NOTE:** It might take some time for the external IP to be provisioned.
+
+### Access the TiKV's PD service
+
+If you run your TiKV cluster behind a LoadBalancer, you can get the external IP of the PD service using the following command:
+
+```shell
+PD_IP=$(kubectl get svc $APP_INSTANCE_NAME-pd \
+  --namespace $NAMESPACE \
+  --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+echo "$PD_IP"
+
+# App metrics
+
+## Prometheus metrics
+
+The app can be configured to expose PD and TiKV metrics 
+in the [Prometheus format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md).
+
+### Configuring Prometheus to collect the metrics
+
+Prometheus can be configured to automatically collect the application's metrics.
+Follow the steps in
+[Configuring Prometheus](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus).
+
+You configure the metrics in the
+[`scrape_configs` section](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
+
+## Exporting metrics to Stackdriver
+
+The deployment includes a
+[Prometheus to Stackdriver (`prometheus-to-sd`)](https://github.com/GoogleCloudPlatform/k8s-stackdriver/tree/master/prometheus-to-sd)
+container. If you enabled the option to export metrics to Stackdriver, the
+metrics are automatically exported to Stackdriver and visible in
+[Stackdriver Metrics Explorer](https://cloud.google.com/monitoring/charts/metrics-explorer).
+The name of each metric starts with the application's name, which you define in
+the `APP_INSTANCE_NAME` environment variable.
+
+The exporting option might not be available for GKE on-prem clusters.
+
+> Note: Stackdriver has [quotas](https://cloud.google.com/monitoring/quotas) for
+> the number of custom metrics created in a single GCP project. If the quota is
+> met, additional metrics might not show up in the Stackdriver Metrics Explorer.
+
+You can remove existing metric descriptors using
+[Stackdriver's REST API](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/delete).
+
+# Scaling up or down
+
+To change the number of TiKV replicas, use the following
+command, where `REPLICAS` is your desired number of replicas:
+
+```shell
+export REPLICAS=3
+kubectl scale statefulset "${APP_INSTANCE_NAME}-tikv" --namespace $NAMESPACE --replicas=$REPLICAS
+```
+
+# Backup and restore
+
+To back up the application, you must back up the database.
+
+### Set up your local environment
+
+Set up environment variables to match your TiKV installation:
+
+```shell
+export APP_INSTANCE_NAME=tikv
+export NAMESPACE=default
+```
+
+## Backing up data
+
+To back up the database, you should use the official tool `br`.
+Use the following commands:
+
+```shell
```

Comment:
You can create a `scripts/` folder as here: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/5c7523aefa0318aadaf3f4b1f809564d335fed85/k8s/mediawiki/scripts/backup.sh

And just provide the reference to the user.

Reply:
well, it's a great idea, i'll do